pkg/build/raw_test.go

```diff
@@ -0,0 +1,89 @@
+package build
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+func TestCreateRawImageCopyCommand(t *testing.T) {
+	// Setup
+	imageConfig := config.ImageConfig{
```

Comment:
I don't love the need for all of this boilerplate setup for each test, but I wasn't looking to address that along with the raw image modification. I'll give it some thought for a separate PR outside of this.